scala/Charts/README.md

```diff
@@ -0,0 +1,12 @@
+# Example JFreeCharts Application (Scala)
+
+This is a sample application which makes a chart based on remote server response.
+The application uses the existing Databricks sample datasets.
+
```

Comment:
Since you're checking in plot.png into the repo, would be nice to embed them into the readme and give a one line description of each.

Something like, 

"The application uses the standard databricks dataset for NYC taxi trips, and produces the following charts.

...

..."

Reply:
Makes sense, including this and two new plots into the README.